### DISCUSSION TITLE
[Bug] Incorrect import path for anchorSelect.css

### DISCUSSION THREAD

**Discussion by corydeppen** [microsoft/monaco-editor#3790] [Q&A] [answered]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [X] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Actual Behavior

I have `monaco-editor@0.33.0` installed in a React app that's using Vite to build files locally and noticed the following error:

```bash
Internal server error: Failed to resolve import "/path/to/node_modules/monaco-editor/esm/vs/editor/contrib/anchorSelect/anchorSelect.css" from "node_modules/.vite/monaco-editor.js?v=659e8648". Does the file exist?

Plugin: vite:import-analysis

File: /path/to/node_modules/.vite/monaco-editor.js?v=659e8648

2995|  init_nls();
2996|  init_contextkey();
2997|  import "/path/to/node_modules/monaco-editor/esm/vs/editor/contrib/anchorSelect/anchorSelect.css";
    |          ^
2998|  var __decorate = function(decorators, target, key, desc) {
2999|    var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
```

In looking for the location of `anchorSelect.css`, I noticed the actual path is `node_modules/monaco-editor/esm/vs/editor/contrib/anchorSelect/browser/anchorSelect.css`.

This error is not seen when I revert to `monaco-editor@0.32.1`.

### Expected Behavior

Should not see the error mentioned when using `monaco-editor@0.33.0`.

**Comment by HakuTheWolfSpirit** [comment, marked as answer]
Hello @corydeppen 
I have stumbled accross this as well. Deleting node_modules/.vite directory and restarting vite server does help for me.
